app/presenters/collection_presenter.rb

```diff
@@ -63,4 +64,51 @@ def license_label
   def review_workflow_status
     collection.review_enabled ? 'On' : 'Off'
   end
+
+  def works
```

Comment:
This all feels like there is a missing view component (instead of being in the presenter).

Reply:
I'll look into extracting a component.